Sources/CLI/Volume/VolumeInspect.swift

```diff
@@ -0,0 +1,40 @@
+//===----------------------------------------------------------------------===//
+// Copyright © 2025 Apple Inc. and the container project authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//===----------------------------------------------------------------------===//
+
+import ArgumentParser
+import ContainerClient
+
+extension Application.VolumeCommand {
+    struct VolumeInspect: AsyncParsableCommand {
+        static let configuration = CommandConfiguration(
+            commandName: "inspect",
+            abstract: "Display detailed information on one or more volumes"
+        )
+
+        @Argument(help: "Volume name(s)")
+        var names: [String]
+
+        @Option(name: .shortAndLong, help: "Format the output using the given Go template")
+        var format: String?
```

Comment:
We should drop this option for a couple reasons:

- Even if there's a suitable Swift version of `pkg/template` we should discuss whether that's the way we want to go for custom formatting.
- In any case it'd be better to implement this later, in a cross-cutting PR so that we don't have one command with it and a bunch of others that should have it but don't.